pkg/kn/commands/revision/revision_list_test.go

```diff
@@ -92,6 +92,23 @@ func TestRevisionListDefaultOutput(t *testing.T) {
 	assert.Check(t, util.ContainsAll(output[2], "bar-wxyz", "bar"))
 }
 
+func TestRevisionListDefaultOutputNoHeaders(t *testing.T) {
+	revision1 := createMockRevisionWithParams("foo-abcd", "foo")
+	revision2 := createMockRevisionWithParams("bar-wxyz", "bar")
+	RevisionList := &v1alpha1.RevisionList{Items: []v1alpha1.Revision{*revision1, *revision2}}
+	action, output, err := fakeRevisionList([]string{"revision", "list", "--no-headers"}, RevisionList)
+	if err != nil {
```

Comment:
We just cleaned up the tests and using
```
	assert.NilError(t, err)
```
it would be good if we follow same approach for tests here.
Check https://github.com/knative/client/pull/264/files for reference

Reply:
done, also cleaned up the other tests in the same files to do the same.